contrib/gorilla/mux/option.go

```diff
@@ -103,3 +106,31 @@ func WithResourceNamer(namer func(router *Router, req *http.Request) string) Rou
 		cfg.resourceNamer = namer
 	}
 }
+
+// WithHeaderTags specifies that the integration should attach HTTP request headers as
+// tags to spans.
+// Warning: using this feature can risk exposing sensitive data such as authorisation tokens
+// to Datadog.
+func WithHeaderTags() RouterOption {
+	return func(cfg *routerConfig) {
+		cfg.headerTags = true
```

Comment:
Why not get rid of `cfg.headerTags` completely and simply append another `StartSpanOption` to `spanOpts`. That's what you are using anyway :)

Reply:
How do I pass the request there?

Reply:
Ah yes, you're right. My bad 🤦